src/utils/getRandomNum.js

```diff
@@ -0,0 +1,8 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
```

Comment:
```suggestion
import { Random } from '@woowacourse/mission-utils';
```
다른 곳은 이렇게 `Random` 객체를 직접 import 하는 경우도 있는데 여기는 `MissionUtils`를 import하는 것 같네요!

성희님만의 기준이 따로 있으신가요??

Reply:
결국 MissionUtils 에 있는 랜덤을 사용하는 것이니 MissionUtils, Random 둘 중 아무거나 import해도 큰 차이가 없다고 생각했습니다!

Reply:
넵 맞습니다! 두 import 방식 모두 같은 동작을 하지만 코드의 일관성 측면에서 하나로 통일하는게 좋다고 생각했는데 어느 곳은 `Random`을 직접 가져오고 여기는 `MissionUtils`을 가져오셔서 특별한 이유가 있었는지 여쭤봤던 것 같습니다🙃